src/extremeweatherbench/utils.py

```diff
@@ -63,24 +63,21 @@ def generate_json_from_nc(u, so, fs, fs_out, json_dir):
             f.write(ujson.dumps(h5chunks.translate()).encode())
 
 
-def clip_dataset_to_bounding_box(
-    dataset: xr.Dataset,
-    location_center: Location,
-    length_km: float,
-) -> xr.Dataset:
-    """Clip an xarray dataset to a boxbox around a given location.
+def get_bounding_corners(
+    location_center: Location, length_km: float, convert_to_360: bool = True
+) -> tuple:
+    """Find corners of bounding box given location center and box size in kilometers.
```

Comment:
On re-reading this, I think it might be better to just request the box size in degrees; the width of a degree longitude depends on what your latitude is. In L83 you use a fixed constant which works okay, but I can imagine scenarios where this is a tad bit too imprecise. If you tweak the `length_km` to `length_degrees` it should be totally fine. You can always add a helper function that accepts length as kilometers, estimates that as degrees, and then calls this function.